css/style.css

```diff
@@ -0,0 +1,903 @@
+.btn-form {
+    margin-top: 33px;
+    width: 244px;
+    height: 53px;
+    font-family: Montserrat, sans-serif;
+    font-size: 18px;
+    line-height: 28px;
+    text-transform: uppercase;
+    letter-spacing: 2px;
+    background-color: #6eeee4;
+    color: rgba(0, 0, 0, .7);
+    border: none;
+    border-radius: 2px;
+    transition: .3s;
+}
+
+.btn-form:hover {
+    transform: scale(1.1);
+    transition: .3s;
+    cursor: pointer;
+    -webkit-transition: .3s;
+    -moz-transition: .3s;
+    -ms-transition: .3s;
+    -o-transition: .3s;
+}
+
+.contact-contact {
+    width: 500px;
+    height: 460px;
+    padding-top: 29px;
+    padding-left: 22px;
+    background-color: rgba(2, 22, 35, .7);
+    box-shadow: 0 0 3px 0 rgb(21 21 21 / 15%);
+}
+
+.contact-adress {
+    padding-left: 8px;
+    margin-bottom: 21px;
+    color: rgba(255, 255, 255, .6);
+    line-height: 24px;
+}
+
+textarea::placeholder,
+input::placeholder {
+    opacity: 0.6;
+    font-family: Raleway;
+    font-size: 15px;
+    line-height: 51px;
+    font-weight: 400;
+    color: #ffffff;
+}
+
+.text-contact:hover {
+    color: rgb(210, 247, 212);
+    transition: .3s;
+    transform: scale(1.05);
+    -webkit-transition: .3s;
+    -moz-transition: .3s;
+    -ms-transition: .3s;
+    -o-transition: .3s;
+    -webkit-transform: scale(1.05);
+    -moz-transform: scale(1.05);
+    -ms-transform: scale(1.05);
+    -o-transform: scale(1.05);
+}
+
+.footer {
+    height: 315px;
+    text-align: center;
+    background-color: #262b2e;
+}
+
+.footer-text {
+    text-align: center;
+    margin-left: 35px;
+    padding-top: 85px;
+}
+
+.footer-text__style {
+    opacity: 0.8;
+    font-family: Raleway;
+    font-size: 16px;
+    line-height: 25px;
+    font-weight: 400;
+    letter-spacing: 0.01em;
+    color: #ffffff;
+    margin-top: 0px;
+    margin-left: -37px;
+}
+
+.social-list {
+    display: flex;
+    justify-content: center;
+    margin-top: 12px;
+}
+
+.social-list_item {
+    width: 46px;
+    height: 40px;
+    background-color: rgba(138, 138, 138, .19);
+    margin: 0 3.5px;
+    list-style-type: none;
+    transition: .3s;
+}
+
+.social-list_link {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.social-list_item i {
+    color: rgba(250, 244, 244, .48);
+    font-size: 22px;
+    transition: .3s;
+    margin-top: 26px;
+    margin-left: 22px;
+    -webkit-transition: .3s;
+    -moz-transition: .3s;
+    -ms-transition: .3s;
+    -o-transition: .3s;
+}
+
+.social-list_item:hover {
+    background-color: #6eeee4;
+    z-index: 1;
+}
+
+.fab:hover {
+    color: rgb(73, 28, 179);
+    z-index: 0;
+}
+
+.text-contact {
+    margin-left: -2px;
+}
+
+.web {
+    margin-top: 26px;
+}
+
+@media (max-width:1200px) {
+    .container {
+        max-width: 960px
+    }
+    .header_image {
+        display: none;
+    }
+    .featured-works__images img {
+        width: 300px;
+        -o-object-fit: cover;

```

Comment:
read about https://github.com/postcss/autoprefixer, if you have to use vendors prefixes

Reply:
thanks, I read the information "Autoprefixer"

Reply:
now try to use it :)